defaults/main.yaml

```diff
@@ -50,3 +50,6 @@ gh_runner_config_labels: ["self-hosted"]
 
 # Set the name to register the runner as in GitHub.
 gh_runner_config_name: "{{ ansible_hostname }}"
+
+# Set the runner group to add the runner to. If left blank, the default runner group will be used.
+gh_runner_config_runnergroup: ""
```

Comment:
Empty/blank default values usually make it harder to understand the role behavior. Requiring people using the role to understand what the expected configuration of a `--runnergroup ""` is.

What about setting the default `runnergroup` directly as the (default) value of `gh_runner_config_runnergroup`?

Reply:
I totally agree and wanted to do that at first. Unfortunately, on the runner groups settings, the default group `Default` can be renamed. If we set the value `default` and if the default group has not be renamed, it works well (it is case insensitive by the way) but if the default group has been renamed, people will hit this error:
```
Could not find any self-hosted runner group named "default"
```
On the other hand, the empty string works well on both use-case.

Reply:
@john-jam oh, thank you for pointing that out. It would then require fetching the default from the API and Ansible wouldn't be "aware" of the default at any point in time since it's not deterministic. 🤔 

Appreciate you clarifying it!